docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,57 @@
+# Mission and Values
+
+This document is meant to help guide decisions about the future of `napari`, be it in terms of
+whether to accept new functionality, changes to the styling of the code or GUI, or whether to take on new dependencies,
+among other things. It serves as a point of reference for [core developers](CORE_DEV_GUIDE.md)
+actively working on the project, and an introduction for newcomers who want to learn a little more
+about where the project is going and what the teams values are. You can also learn more about how the project is managed by looking at our [governance model](GOVERNANCE).
+
+## Our mission
+
+napari aims to be the **multi-dimensional image viewer for Python** and to **provide GUI access to a plugin ecosystem of image analysis tools for scientists** to use in their daily work. We hope to accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new dependencies, sometimes making them optional, and will support a fully packaged installation that works cross-platform.
+
+- being **well documented** with **comprehensive tutorials and examples**. All functions in our API have thorough docstrings clarifying expected inputs and outputs, and we maintain a separate [tutorials and example website](http://napari.org) to explain different use cases and working modes.
+
+- providing **GUI access** to all critical functionality so napari can be used by people with no coding experience.
+
+- being **interactive** and **highly performant** in order to support very large data sets.
```

Comment:
This is the first value, imho.

I think this document should list napari's founding principles:

- n-dimensional first;
- native (no browser required);
- blazing fast (GPU-powered);
- native 2D and 3D viewing; and
- minimal new windows and pop-ups, choosing layering instead.

Feel free to add caveats for the "no browser" clause (I know that many users want to use napari within the browser), but I think "no browser **required**" captures the principle well while allowing for in-browser development in the future.

Reply:
yup these are great, i'll look at how best to integrate them (i.e. separate list, call out as "founding principles", integrated in other places). Might take more than one attempt, but 100% agree with them

Reply:
yes, I would love to see these founding principles somewhere.
of course, we may have new principles that come afterwards... ;-)
I think, since this file is 'Mission and values' it does fit there perhaps.
Perhaps a separate file 'FOUNDING PRINCIPLES' or 'PHYLOSOPHY'
could capture some of the high level concepts, and could become a bit
of a tutorial on the design principles that we have adopted, from high level
founding principles, to low level consideration sof model/view, testing, and 
other such things that we have been doing either explicitly or implicitly...  

Could also live in the core dev guide... hmm...

Reply:
i've added a 'founding principles' section - I think it works, but happy to make changes too